Source/Scene/Cesium3DTileset.js

```diff
@@ -1053,25 +1053,35 @@ define([
     }
 
     function getScreenSpaceError(tileset, geometricError, tile, frameState) {
-        // TODO: screenSpaceError2D like QuadtreePrimitive.js
         if (geometricError === 0.0) {
             // Leaf nodes do not have any error so save the computation
             return 0.0;
         }
 
         // Avoid divide by zero when viewer is inside the tile
         var camera = frameState.camera;
-        var distance = Math.max(tile.distanceToCamera, CesiumMath.EPSILON7);
-        var height = frameState.context.drawingBufferHeight;
-        var sseDenominator = camera.frustum.sseDenominator;
+        var context = frameState.context;
+        var height = context.drawingBufferHeight;
 
-        var error = (geometricError * height) / (distance * sseDenominator);
+        var error;
+        if (frameState.mode === SceneMode.SCENE2D) {
+            var frustum = camera.frustum;
+            var width = context.drawingBufferWidth;
 
-        if (tileset.dynamicScreenSpaceError) {
-            var density = tileset._dynamicScreenSpaceErrorComputedDensity;
-            var factor = tileset.dynamicScreenSpaceErrorFactor;
-            var dynamicError = CesiumMath.fog(distance, density) * factor;
-            error -= dynamicError;
+            var pixelSize = Math.max(frustum.top - frustum.bottom, frustum.right - frustum.left) / Math.max(width, height);
+            error = geometricError / pixelSize;
+        } else {
+            // Avoid divide by zero when viewer is inside the tile
+            var distance = Math.max(tile.distanceToCamera, CesiumMath.EPSILON7);
+            var sseDenominator = camera.frustum.sseDenominator;
+            error = (geometricError * height) / (distance * sseDenominator);
+
+            if (tileset.dynamicScreenSpaceError) {
```

Comment:
I don't think it would make sense in 2D. Maybe CV, but we don't have fog in CV either.